README.md

```diff
@@ -151,6 +151,16 @@ def admin_list
 end
 ```
 
+It is important to note that `authorize` returns the object passed to it,
```

Comment:
I disagree that it is "important". I'd maybe say "kinda neat", but not "important" ;).